react/MiniCart.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { graphql } from 'react-apollo'
+import orderFormQuery from './graphql/orderFormQuery.gql'
+import MiniCartItem from './MiniCartItem'
+import Button from '@vtex/styleguide/lib/Button'
+import { Price } from '@vtex/product-details'
+import Spinner from '@vtex/styleguide/lib/Spinner'
+import './global.css'
+
+/**
+ * Minicart component
+ */
+class MiniCart extends Component {
+  handleClickButton = () => location.assign('/checkout/#/cart')
+
+  renderWithoutProducts = (label) => (
+    <div className="minicart-item pa4 shadow-4 flex items-center justify-center">
+      <span className="f5">{label}</span>
+    </div>
+  )
+
+  renderMiniCartWithItems = (orderForm, label) => {
+    let contentStyle = 'pa4'
+    if (orderForm.items.length > 3) {
+      contentStyle += ' overflow-auto minicart-content mb4'
```

Comment:
`contentStyle` -> `contentClass`
`minicart-content` -> `vtex-minicart__content` e também não precisa desse `if`

Faz assim:

```
let contentStyle = 'vtex-minicart__content overflow-auto pa4 mb4'
```
daí no css troca isso `height: 300px;` por `max-height: 300px;`

Dessa maneira a classe `vtex-minicart__content` sempre vai estar presente no escopo e não só quando o num de items for maior que 3

Reply:
Done!

Reply:
Done!